wd_core.au3

```diff
@@ -1222,7 +1222,7 @@ Func _WD_Startup()
 	$iPID = ProcessExists($sFile)
 
 	If $_WD_DRIVER_DETECT And $iPID Then
-		__WD_ConsoleWrite($sFuncName & ": Existing instance of " & $sFile & " detected!")
```

Comment:
I also wonder if this should be changed to not show only when `_WD_DEBUG>=_WD_DEBUG_Info`
```autoit
__WD_ConsoleWrite($sFuncName & ": Existing instance of " & $sFile & " ( PID=" & $iPID & " ) detected!", $_WD_DEBUG_Info)

```

Reply:
Good idea. See https://github.com/Danp2/au3WebDriver/pull/327/commits/8a6ed0c73af09bfef9b4d3dc1a7eb64a0ea41c09